src/api/app/models/user.rb

```diff
@@ -349,6 +349,7 @@ def password=(value)
   def update_password(pass)
```

Comment:
this update_password is only used for api users right? wouldn't it make sense to add it to b694d37e6b5bb0f84b741e6557c3ad0cac7fb8a7